test/parallel/test-tls-server-verify.js

```diff
@@ -20,6 +20,10 @@
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 'use strict';
+
+// If this issue gets fixed on Windows CI, this test can replace
+// test/parallel/test-tls-server-verify.js once the 60000 ms timer is removed.
```

Comment:
Should this be referencing the version in known_issues?

Reply:
No, and it won't when (if) I finished. It's a work-in-progress. I copied over the test to get an idea of what fails and how in CI.